db/migrate/20191029022906_populate_is_mix_on_playlist.rb

```diff
@@ -0,0 +1,11 @@
+class PopulateIsMixOnPlaylist < ActiveRecord::Migration[6.0]
+  def change
+    counter = 0
+    Playlist.all.map do |playlist|
+      playlist.set_mix_or_album
```

Comment:
I think you removed / renamed this method, does it need to change here?

Reply:
good catch!